tests/APIv2/ThemesTest.php

```diff
@@ -82,4 +84,32 @@ public function testGetByName() {
             $this->assertArrayHasKey($asset, $body["assets"], "Theme does not have expected asset: {$asset}");
         }
     }
+
+    /**
+     * Test getting a theme's logo.
+     *
+     * @depends testGetByName
+     */
+    public function testLogo() {
+        $logo = "logo.png";
+        self::container()->get(Gdn_Configuration::class)->set("Garden.Logo", $logo);
```

Comment:
I don't know if you're aware but we do have a `MockConfig` object.

See usage https://github.com/vanilla/vanilla/blob/d79e5924f7d0ccc7df8f7e27ad65d44ad892c947/tests/Library/Vanilla/Formatting/GdnFormatLinksTest.php#L35

Reply:
This would be great for tests with minimal config requirements. Unfortunately, our API integration tests require a lot of additional keys to get the site up and running and in a state where it can respond to API requests. Using `MockConfig` for API/integration tests is probably something we could look into doing in the future, but would require some more groundwork that's outside the scope of this PR.